noetic/distribution.yaml

```diff
@@ -1091,14 +1091,30 @@ repositories:
     doc:
       type: git
       url: https://github.com/4am-robotics/cob_common.git
-      version: kinetic_release_candidate
+      version: noetic-release-candidate
```

Comment:
It is uncommon for the `doc` version and the `source` version to be different.  It isn't strictly disallowed, but can you explain why these are different?